.travis.yml

```diff
@@ -96,6 +96,7 @@ matrix:
         apt:
           packages:
             - openjdk-8-jre
+            - netcat
```

Comment:
actually won't need netcat anywhere... gonna wait for CI to complete, and then delete this and run CI again